zookeeper-server/src/main/java/org/apache/zookeeper/jmx/ManagedUtil.java

```diff
@@ -71,7 +71,7 @@ public static void registerLog4jMBeans() throws JMException {
                 // org.apache.log4j.jmx.HierarchyDynamicMBean hdm = new org.apache.log4j.jmx.HierarchyDynamicMBean();
                 Object hdm = Class.forName("org.apache.log4j.jmx.HierarchyDynamicMBean").getDeclaredConstructor().newInstance();
 
-                ObjectName mbo = new ObjectName("log4j:hiearchy=default");
+                ObjectName mbo = new ObjectName("log4j:hierarchy=default");
```

Comment:
Is change may have an impact on integrated systems.
Maybe we could make this name configurable so that people will be able to revert the name.
This change can be only in master (3.6.0) as it is a public configuration/API change

Reply:
Hi, @eolivelli . Thanks for your reviewing. Indeed, turning it into a configurable one would be very helpful in upgrading an integrated system. So, let's be clear, should we add a new JVM option or add a new environment variable to the operating system?